### DISCUSSION TITLE
Technitium - DHCP Server

### DISCUSSION THREAD

**Discussion by roswitina** [TechnitiumSoftware/DnsServer#874] [Q&A] [answered]
I intend to test Technitium DNS on my internal network. I have not yet had any experience with Technitium, but I have already read up on it a bit. Among other things, one question has remained unanswered.
Is it possible to switch the DHCP server on and off separately or is it started with the DNS server and must be running?

**Comment by ShreyasZare** [comment, marked as answer]
Thanks for asking. The DHCP server will start only when there is a DHCP scope that is enabled. If no DHCP scope is enabled then the DHCP server will stopped.

**Reply by ylluminate** [reply]
This does not seem intuitive. It seems that there should be a more obvious enable/disable function for DHCP service vs creating and destroying scopes... We really need to have a more clear view on this being on/off or otherwise.

**Reply by ShreyasZare** [reply]
@ylluminate Thanks for the feedback. There is really no "on/off" thing for DHCP server. The DHCP server will listen on udp port for the configured scope and when there is no scope, there is literally nothing to do there. Adding an artificial "on/off" option would make it complex by needing to configure one more option and for many users, it will not be intuitive.

**Reply by Rawsted** [reply]
> @ylluminate Thanks for the feedback. There is really no "on/off" thing for DHCP server. The DHCP server will listen on udp port for the configured scope and when there is no scope, there is literally nothing to do there. Adding an artificial "on/off" option would make it complex by needing to configure one more option and for many users, it will not be intuitive.

could someone explain it in step by step like: Go to settings, go to General,... and so on please i don´t get it chatgpt doesn´t get it

**Reply by ShreyasZare** [reply]
@Rawsted I am not sure what exactly you are looking for. The DHCP Server is configured in the DHCP tab on the admin panel. You just need to ensure that the server has a static IP address and then create a DHCP Scope that uses the start and end IP address in the same network range as that of the static IP.
